crates/core/src/models/mod.rs

```diff
@@ -24,6 +33,14 @@ pub struct TokenBalances {
     pub token_balances: Vec<TokenBalance>,
 }
 
+#[derive(Debug, Error)]
+pub enum ConversionError {
+    #[error("transaction conversion error: {0}")]
+    TransactionConvertionError(String),
```

Comment:
typo "Conver**t**ion"

Reply:
I think it's also conversion in english no?

Reply:
this => `TransactionConvertionError` should be `TransactionConversionError`